Failed test on go 1.15

<img src="https://raw.githubusercontent.com/derailed/k9s/master/assets/k9s_err.png" align="right" width="100" height="auto"/>

<br/>
<br/>
<br/>


**Describe the bug**
Failed test: k9s/internal/ui on Golang 1.15

**To Reproduce**
1. `go build && go test ./...`

**Expected behavior**
No failed test

**Outputs**

```
 # github.com/derailed/k9s/internal/ui
internal/ui/action.go:86:57: conversion from int to string yields a string of one rune, not a string of digits (did you mean fmt.Sprint(x)?)
?   	github.com/derailed/k9s	[no test files]
?   	github.com/derailed/k9s/cmd	[no test files]
?   	github.com/derailed/k9s/internal	[no test files]
ok  	github.com/derailed/k9s/internal/client	0.057s
ok  	github.com/derailed/k9s/internal/color	0.008s
ok  	github.com/derailed/k9s/internal/config	0.055s
ok  	github.com/derailed/k9s/internal/dao	0.062s
ok  	github.com/derailed/k9s/internal/health	0.008s
ok  	github.com/derailed/k9s/internal/model	2.103s
?   	github.com/derailed/k9s/internal/perf	[no test files]
ok  	github.com/derailed/k9s/internal/render	0.143s
ok  	github.com/derailed/k9s/internal/tchart	0.007s
FAIL	github.com/derailed/k9s/internal/ui [build failed]
```
**Versions (please complete the following information):**
 - OS: Linux
 - K9s: 0.21.7
 - K8s: 1.18.0

**Additional context**
Golang 1.15

--- a/.travis.yml
+++ b/.travis.yml
@@ -1,7 +1,7 @@
 language: go
 go_import_path: github.com/derailed/k9s
 go:
-  - "1.13"
+  - "1.15"
 
 os:
   - linux

--- a/internal/ui/action.go
+++ b/internal/ui/action.go
@@ -83,7 +83,7 @@ func (a KeyActions) Hints() model.MenuHints {
 				},
 			)
 		} else {
-			log.Error().Msgf("Unable to locate KeyName for %#v", string(k))
+			log.Error().Msgf("Unable to locate KeyName for %#v", string(rune(k)))
 		}
 	}
 	return hh

